highlight.io/components/Launch/Week3/DayFive.tsx

```diff
@@ -23,23 +25,44 @@ const DayFive = () => {
 
 	return (
 		<div id="day-5" className="w-full max-w-[550px] md:max-w-none z-50">
-			<div
-				onClick={() => handleCopy('#day-5')}
-				className="group flex items-center gap-2 cursor-pointer"
-			>
-				<Typography
-					className="text-darker-copy-on-dark"
-					type="copy3"
-					emphasis
+			<div className="flex justify-between items-center">
+				<div className="group flex items-center gap-2 w-1/2 cursor-pointer">
+					<Typography
+						className="text-darker-copy-on-dark"
+						type="copy3"
+						emphasis
+					>
+						Day 5: October 20th
+					</Typography>
+
+					<a
+						className="flex items-center gap-1 border border-[#34343A] text-darker-copy-on-dark px-4 py-1 rounded-full"
+						href="https://drive.google.com/file/d/1D13VqO4u4QfYq1Lz8xIIpd_OIjAddkYf/view?usp=sharing"
```

Comment:
shouldn't we be linking to the https://www.youtube.com/@highlight-io/videos videos @Abhishek-More ?

Reply:
O, I didn't know this existed. I asked Chris about it and he sent me some google drive links. Will change now

Reply:
Changed

Reply:
ty!